### PR TITLE
FA-95 createAccountCompose Module

### PR DESCRIPTION
**Pull request by brosscode**
Adds a createAccount Compose Module.

### CODE REVIEW

File: createAccountCompose/src/androidTest/java/com/brandon/createaccount/compose/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.brandon.createaccount.compose
```
Comment (ForzenAcademy) [inline createAccountCompose/src/androidTest/java/com/brandon/createaccount/compose/ExampleInstrumentedTest.kt]: Remove

File: loginCompose/src/androidTest/java/com/brandon/logincompose/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.brandon.logincompose
```
Comment (ForzenAcademy) [inline loginCompose/src/androidTest/java/com/brandon/logincompose/ExampleInstrumentedTest.kt]: Remove

**Review by ForzenAcademy — APPROVED** [review]

File: createAccountCompose/src/androidTest/java/com/brandon/createaccount/compose/ExampleInstrumentedTest.kt (hunk 1 above)
Comment (brosscode) [reply createAccountCompose/src/androidTest/java/com/brandon/createaccount/compose/ExampleInstrumentedTest.kt]: Removed

File: loginCompose/src/androidTest/java/com/brandon/logincompose/ExampleInstrumentedTest.kt (hunk 2 above)
Comment (brosscode) [reply loginCompose/src/androidTest/java/com/brandon/logincompose/ExampleInstrumentedTest.kt]: Removed

**Pull request merged**